base/applications/osk/main.c

```diff
@@ -138,19 +138,14 @@ DWORD WINAPI OSK_WarningDlgThread(LPVOID lpParameter)
 VOID OSK_About(VOID)
 {
     WCHAR szAuthors[MAX_PATH];
-    HICON OSKIcon;
-
-    /* Load the icon */
-    OSKIcon = LoadImageW(Globals.hInstance, MAKEINTRESOURCEW(IDI_OSK), IMAGE_ICON, 0, 0, LR_DEFAULTSIZE);
 
     /* Load the strings into the "About" dialog */
     LoadStringW(Globals.hInstance, IDS_AUTHORS, szAuthors, _countof(szAuthors));
 
+    /* Load the icon */
     /* Finally, execute the "About" dialog by using the Shell routine */
-    ShellAboutW(Globals.hMainWnd, Globals.szTitle, szAuthors, OSKIcon);
-
-    /* Once done, destroy the icon */
-    DestroyIcon(OSKIcon);
+    ShellAboutW(Globals.hMainWnd, Globals.szTitle, szAuthors,
+                LoadImageW(Globals.hInstance, MAKEINTRESOURCEW(IDI_OSK), IMAGE_ICON, 0, 0, LR_DEFAULTSIZE | LR_SHARED));
```

Comment:
I don't get the point of this. `LoadImageW` in the original will load the icon each time a call is invoked to it, thus a uniquely handle is given for it. It is not a shared icon so it is correct to call `DestroyIcon` so I don't get what are you trying to achieve here.

Reply:
`LR_SHARED` does not give a unique handle every time,
as it implies: it is a shared resource.

Reply:
GeoB99, yes, this case existing code is self-consistent. I am simply switching it to the same shared pattern as the other calls.

Reply:
I'm not a fan of randomly changing code on stuff that actually is fine, the ticket on itself says about that there might be calls that create shared images and then they're freed afterwards, when in OSK this is not actually the case. But fine, you get a pass.